src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryMultiSink.java

```diff
@@ -121,7 +122,8 @@ protected void prepareRunInternal(BatchSinkContext context, BigQuery bigQuery, S
    * This method sanitizes outputName when there is an un-allowed special character in dataset
    * For example: If we have a dataset with outputName testtable$2020, this method will sanitize it to testtable_2020
    */
-  private String sanitizeOutputName(String outputName) {
+  @VisibleForTesting
+  protected String sanitizeOutputName(String outputName) {
```

Comment:
Does it need to be protected, just use default instead?

Reply:
PR updated.